.github/workflows/release.yml

```diff
@@ -13,14 +13,14 @@ jobs:
       - name: Setup Sage
         uses: einride/sage/actions/setup@master
         with:
-          go-version: "1.23"
+          go-version: "1.24"
 
       - name: Make
         run: make
 
       - name: Release
         id: semantic-release
-        uses: go-semantic-release/action@v1.24
```

Comment:
I'm not sure why this is `@v1.24` here. Seems like a mistake. Reference: https://github.com/go-semantic-release/semantic-release#examples

Reply:
Dependabot is handling that version: [config](https://github.com/einride/spanner-aip-go/blob/a1069afe61e22112a5b0af7a0ca9ba1e9767b431/.github/dependabot.yml#L4-L7), [example bump](https://github.com/einride/spanner-aip-go/commit/cf53e9e56db7f1e12f63a77753e02f092e11c5d2)

Reply:
Ok, should've blamed that of course. Their README states @v1 which would give us the latest v1.x. So this change would, if we keep it in, reduce Dependabot noice but maybe give us less stability. 🤷  I can roll that change back if you think it's better. 👍